src/styles/_variables.scss

```diff
@@ -1,26 +1,29 @@
-$primary-color: #1DA1F2;
-$secondary-color: #14171A;
-$background-color: #FFFFFF;
-$text-color: #14171A;
-$gray-light: #E1E8ED;
+$primary-color: #1da1f2;
+$secondary-color: #14171a;
+$background-color: rgba(182, 178, 178, 0.1);
+$gray-light: #e1e8ed;
 $gray-dark: #657786;
-$error-color: #E0245E;
-$success-color: #17BF63;
+$error-color: #e0245e;
+$success-color: #17bf63;
+$black: #000000;
+$border-color: rgba(182, 178, 178, 0.1);
+$text-color: #ffffff;
+$input-background-color: rgba(182, 178, 178, 0.01);
 
 // Font sizes
-$font-xs: 0.75rem;    // 12px
-$font-sm: 0.875rem;   // 14px
-$font-base: 1rem;     // 16px
-$font-lg: 1.125rem;   // 18px
-$font-xl: 1.25rem;    // 20px
-$font-2xl: 1.5rem;    // 24px
+$font-xs: 0.75rem; // 12px
+$font-sm: 0.875rem; // 14px
+$font-base: 1rem; // 16px
+$font-lg: 1.125rem; // 18px
+$font-xl: 1.25rem; // 20px
+$font-2xl: 1.5rem; // 24px
```

Comment:
as i mentioned earlier as well you should be using these font size variables not hard coded values like you did in the code. and if you are seeing any values in designs like 11 or 10px instead of 12px you should use 12px not hard coded 11 or 10

Reply:
fixed font-size from hard coded to variables